src/transformers/models/efficientnet/image_processing_efficientnet.py

```diff
@@ -153,26 +160,25 @@ def rescale(
         **kwargs,
     ):
         """
-        Rescale an image by a scale factor. image = image * scale.
+        Rescale an image by a scale factor.
+
+        If offset is `True`, image scaled between [-1, 1]: image = (image - scale / 2) * scale. If offset is `False`,
+        image scaled between [0, 1]: image = image * scale
 
         Args:
             image (`np.ndarray`):
                 Image to rescale.
             scale (`int` or `float`):
                 Scale to apply to the image.
-            offset (`bool`, *optional*):
+           offset (`bool`, *optional*):
```

Comment:
seems missing a space ?

Reply:
Yes - the irony that it's for the `offset` argument is not lost on me 🙈